resources/js/main.js

```diff
@@ -0,0 +1,287 @@
+import ithenticateSimilarityScoreCell from "./Components/ithenticateSimilarityScoreCell.vue";
+
+pkp.registry.registerComponent("ithenticateSimilarityScoreCell", ithenticateSimilarityScoreCell);
+
+const {useLocalize } = pkp.modules.useLocalize;
+const { useApp } = pkp.modules.useApp;
+const { useNotify } = pkp.modules.useNotify;
+import { computed, watch } from "vue";
+import deduceFileStatus from "./fileStatus";
+
+function runPlagiarismAction(piniaContext, stageNamespace) {
+    
+    const { useUrl } = pkp.modules.useUrl;
+    const { useFetch } = pkp.modules.useFetch;
+    const { t } = useLocalize();
+
+    const { isOPS } = useApp();
+
+    const fileStore = piniaContext.store;
+    const { submission, submissionStageId } = fileStore.props;
+
+    const ithenticateQueryParams = computed(() => {
```

Comment:
Since its POST with payload.. probably would not call it query params.  My example was actually expecting that the list of files will be passed as query param in the url.

Reply:
we can change it something like `ithenticateRequestParams` , but really not that important as request can be changed to GET also . Updated it anyway.